### PR TITLE
[FIX] travis

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
:+1:

**Comment by sbidoul** [comment]
:+1:

File: account_banking/__openerp__.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 ##############################################################################
 {
     'name': 'Account Banking',
-    'version': '0.5',
+    'version': '7.0.0.0.5',
```
Comment (elicoidal) [inline account_banking/__openerp__.py:28]: usually 7.0.1.0.5 no?

**Comment by pedrobaeza** [comment]
As this is an old one branch, we are not going to care too much for the version number, so I merge.

**Pull request merged**